src/main/java/hudson/plugins/build_timeout/impl/DeadlineTimeOutStrategy.java

```diff
@@ -20,7 +20,7 @@
 /**
  * If the build reaches <tt>deadlineTime</tt>, it will be terminated.
  * 
- * @author Fernando Migu�lez Palomo <fernando.miguelez@gmail.com>
```

Comment:
I found that this was not UTF-8. This should be 0xc3a9 in UTF-8, but was 0xe9 (supposed latin-1).